### ISSUE TITLE
Release 2.0.0 YAMLParseError

### ISSUE THREAD

**Issue by hauselin** [beaussan/update-time-on-edit-obsidian#48]
The latest release 2.0.0 doesn't work well with Obsidian's beta Properties feature. I'm getting a lot of `YAMLParseErrors` in the console, and is preventing Obsidian from even starting. 

```
Uncaught (in promise) YAMLParseError: Implicit keys need to be on a single line at line 1, column 1:
Uncaught (in promise) YAMLParseError: Unexpected flow-map-start at node end at line 4, column 20:
```

**Comment by beaussan** [comment]
It's working well on my side with the Properties features, however, it seems a file has malformed yaml in your vault, I will add a error clause to prevent the plugin from craching

**Comment by beaussan** [comment]
This should be fixed with the 2.0.1 version :tada:
Feel free to reopen an issue if this is still an issue
